problemsets-mat223/modules/module1.tex

```diff
@@ -423,7 +423,7 @@
 	\end{tikzpicture}
 \end{center}
 
-In conjunction with the standard coordinate system, there are also \emph{standard basis vectors}.
+In conjunction with the standard coordinate system, there are also \emph{standard basis vectors}\index[definitions]{Standard basis}\index{Standard basis}\index[definitions]{Basis!standard basis}\index{Basis!standard basis}.
```

Comment:
This isn't really where the standard basis is defined...it is only mentioned here and defined in the basis chapter.